jaxrs-scala-clients/build.gradle

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'com.github.johnrengelman.shadow'
+apply from: "${rootDir}/gradle/publish-shadow.gradle"
+
+dependencies {
+    compile project(":jaxrs-clients")
+    compile "com.fasterxml.jackson.module:jackson-module-scala_2.11"
+
+    // Keep in sync with the dependencies excluded from shadowJar.
+    shadow project(":jaxrs-clients")
+    shadow "com.fasterxml.jackson.module:jackson-module-scala_2.11"
+}
+
+shadowJar {
```

Comment:
:( this is pretty unmaintainable already, but duplicating this logic into two places makes it worse. can we at least factor out?